addons/l10n_es/data/account_tax_data.xml

```diff
@@ -2080,12 +2110,12 @@
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'base',
-                'tag_ids': [ref('mod_303_01')],
+                'tag_ids': [ref('mod_303_153')],
             }),
             (0,0, {
                 'factor_percent': 100,
                 'repartition_type': 'tax',
-                'tag_ids': [ref('mod_303_03')],
+                'tag_ids': [ref('mod_303_155')],
```

Comment:
You'll need to update the taxes configuration with a local migration script, and change the module version number, just like Claire did with LU here https://github.com/odoo/odoo/pull/108571/files (brand new feature, but let's use it ! :) )

Reply:
@oco-odoo I added the migration script using Claire's magic function :magic_wand:

Reply:
@oco-odoo Migration script has been added and tested. Meanwhile fixed a missed case in the migration script as well.

Reply:
Nice catch, thanks :)